prov/tcp/src/tcpx.h

```diff
@@ -246,6 +246,7 @@ struct tcpx_fabric {
 	struct util_fabric	util_fabric;
 };
 
+#define TCXP_INTERNAL_XFER	BIT_ULL(60)
```

Comment:
TXPC?

Reply:
gah -- yes -- I'm sooooooo over typing tcpx_ at this point.  It's an extreme finger exercise.  Will update

Reply:
actually, in my comment... I typed it wrong too... hah. had to edit :P